src/test/java/AndreeaG/tests/tema5/Demoqa/Buttons.java

```diff
@@ -0,0 +1,36 @@
+package AndreeaG.tests.tema5.Demoqa;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import teofilursan.driver.BrowserManager;
+
+public class Buttons {
+
+
+    public static void main(String[] args) {
+     clickButtons();
+
+    }
+
+    public static void clickButtons(){
+        ChromeDriver driver = BrowserManager.createChromeDriver();
+        driver.get("https://demoqa.com/buttons");
+        WebElement doubleClickMe = driver.findElement(By.id("doubleClickBtn"));
+        doubleClickMe.click();
+        System.out.println("Clicked on Double Click me button and text shown is: " +doubleClickMe.getText());
+
+        WebElement rightClickMe = driver.findElement(By.id("rightClickBtn"));
+        rightClickMe.click();
+        System.out.println("Clicked on Right Click me button the text shows: " +rightClickMe.getText());
+
+        WebElement clickMeClickMe = driver.findElement(By.xpath("/html/body/div[2]/div/div/div[2]/div[2]/div[2]/div[3]/button"));
```

Comment:
Replace the absolute xpaths with relative xpaths. Applicable for all classes that use absolute xpaths. You can find some example in other PRs too. Thanks!

Reply:
I used : //div[3]/button instead.

ty!

Reply:
replaced all absolute xpaths (I think :)